### PR TITLE
migrating frax_finance bnb to dunesql

### PR DESCRIPTION
**Pull request by Vahid-flipside**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by Vahid-flipside** [comment]
Have a hard time understanding the error. 
Given that my config seems right!!

**Comment by Vahid-flipside** [comment]
@Hosuke Is there anything I should do for this?

**Review by jeff-dude — CHANGES_REQUESTED** [review]
looks good for the most part, thank you for helping migrate.

one minor callout:
can you ensure all column names / cte names / table names / etc (and corresponding changes to schema.yml file to reflect it) are set to lower and underscore format?
`some_column_name` for example

**Comment by Vahid-flipside** [comment]
> 

Done

File: models/frax_finance/bnb/frax_finance_bnb_bribes.sql [hunk 1]
```diff
@@ -415,58 +472,80 @@ fee_weekly_rewards as (
...
-    round(c.frax_TVL) as end_frax_tvl,
-    round(b.frxETH_TVL) as start_frxETH_tvl,
-    round(c.frxETH_TVL) as end_frxETH_tvl
+    round(g.fee_reward_token0) as fee_reward_token0,
+    round(g.fee_reward_token1) as fee_reward_token1,
+    d.FrxETH_bribe as lwb_frxETH,
+    round(d.FrxETH_bribe * h.price) as lwb_frxETH_usd,
+    b.FrxETH_reserve as start_frxETH,
+    c.FrxETH_reserve as end_frxETH,
+    round(b.FrxETH_TVL) as start_frxETH_TVL,
```
Comment (jeff-dude) [inline models/frax_finance/bnb/frax_finance_bnb_bribes.sql:512]: it looks like there are still instances throughout, like this section for example

Comment (Vahid-flipside) [reply models/frax_finance/bnb/frax_finance_bnb_bribes.sql:512]: I guess its how frxETH is written every where. I haven't capitalized it. 
Cause you won't see frxeth on their website either.

Comment (jeff-dude) [reply models/frax_finance/bnb/frax_finance_bnb_bribes.sql:512]: its just a spellbook specific database naming standard. for instance, lido uses lowercase on `stETH` to `steth`, such as here:
https://github.com/duneanalytics/spellbook/blob/main/models/lido/liquidity/ethereum/lido_liquidity_ethereum_curve_steth_conc_pool.sql#L28

Comment (Vahid-flipside) [reply models/frax_finance/bnb/frax_finance_bnb_bribes.sql]: @jeff-dude Noted. 
It's fixed now.

**Review by jeff-dude — APPROVED** [review]
thank you!

**Comment by jeff-dude** [comment]
ah, we had a recent merge, so your PR picked up unintended models/tests to run. rerunning now should resolve that.

**Pull request merged**